src/components/hero.js

```diff
@@ -1,30 +1,42 @@
 /** @jsx jsx */
 import { Box, jsx } from 'theme-ui'
 
-const Hero = ({ children, big, ...props }) => (
+import CircuitBoard from '../images/circuit-board.svg'
```

Comment:
We may want to setup something like absolute paths so we don't have to go through a bunch of nested relative directories for importing

Reply:
meh, one .. is not too bad I think

Reply:
yeah definitely not bad at all lol, not sure how the routing works for Gatsby but in nextjs it was all nested dirs